src/sagemaker/lineage/query.py

```diff
@@ -325,23 +381,51 @@ def render(self, elements, path="pyvisExample.html"):
             display graph: The interactive visualization is presented as a static HTML file.
 
         """
-        net = self.Network(height="500px", width="100%", notebook=True, directed=True)
+        net = self.Network(height="600px", width="82%", notebook=True, directed=True)
```

Comment:
How were these numbers created? `82%` especially pops out a bit. Does it break/look worse at `80%`?

Reply:
82% gives the pyvis graph max area to display, 80% doesn't break or look very different

Reply:
Nope, just curious.